types/webextension-polyfill/namespaces/theme.d.ts

```diff
@@ -45,7 +40,7 @@ export namespace Theme {
          *
          * @param windowId Optional. The window for which we want the theme.
          */
-        getCurrent(windowId?: number): Promise<any>;
```

Comment:
I noticed there are a bunch of places where `any` was changed to `unknown`. In input positions that should be fine, but in an output position like here in the return type of a function, it's possible that this change will break users. Is there a reason for this kind of change now?

Reply:
Hi @gabritto,

thanks for the review. You're right, it will be a breaking change for some users. I would argue though, that it's worth the effort, as an unknown type is better than an any type. With any you can't easily spot mistakes. With unknown, you'll have to make sure what the type is.

Usually these unknown types in webextensions are  responses from a message handler, storage, a script execution or similar, where only the caller actually knows what might be returned and thus should specify the type manually.

In that specific case though, you've spotted a good case where it should be neither. It should actually be a `Theme` here. I'll have to create a manual fix for this one, as the mozilla schema files don't give away the correct type.

I've just checked again and could not find anything aside from `getCurrent` that is not in the responsibility of the caller. If you see anything else, please let me know.

I'll push an update soon.

Thanks.

Reply:
I've pushed the fix and a separate commit for other schema updates that have appeared upstream in the meantime.